src/pages/Home/Menu/MenuModal.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Categories } from "../../../DummyData/Categories";
+import CategoryLi from "./Category/CategoryLi";
+import { MenuModalStyle } from "./MenuModalStyle";
+
+type Props = {};
+
+export default function MenuModal() {
+  return (
+    <MenuModalStyle>
+      <div className="handle" />
```

Comment:
use styled-component instead of className. Create a new styled component for this element